src/dsp_tools/utils/connection_live.py

```diff
@@ -195,15 +198,10 @@ def get(
         if not route.startswith("/"):
             route = f"/{route}"
         url = self.server + route
-        if not headers:
-            headers = {}
```

Comment:
These are still in post and put, is there a reason why we don't have headers here?

Reply:
The only header we had here was the "User-Agent", which is now set in the `__post__init__()`. So there is no semantic change here, the User-Agent is still sent.